File: tests/fixtures/php/calls.php
<?php
class Worker
{
    public static function create(): self { return new self(resolve()); }
}

function run(Service $service, callable $loader): void
{
    $loader();
    helper();
    Service::create();
    $service->execute();
    $service?->maybe();
    factory()->build();
    new Worker(build());
}

function helper(): void {}

File: tests/fixtures/scala/calls.scala
object Calls:
  def run(loader: () => String, service: Service, head: String, items: List[String]): Unit =
    loader()
    helper(1)(2)
    service.execute()
    Type.create[String]()
    items map process
    head :: items
    new Worker(build())
    summon[Ordering[String]]

  def helper(first: Int)(second: Int): Int = first + second

class Worker(value: String)
